rhizome/host/bin/install-liburing

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Spdk requires liburing 2.2. The liburing available in apt repositories
+# for Ubuntu 22.04 is 2.1, so we need to install it from source.
+
+GIT_REPO_LIBURING=https://github.com/axboe/liburing.git
+liburing_dir=/usr/local/src/liburing
+
+if [[ -d $liburing_dir ]]; then
+    echo "liburing source already present, not cloning"
+else
+    mkdir -p $liburing_dir
+    git clone "${GIT_REPO_LIBURING}" "$liburing_dir"
```

Comment:
nit: we can clone just given tag with a single revision.
```suggestion
    git clone --depth 1 --branch liburing-2.2 "${GIT_REPO_LIBURING} "$liburing_dir"
```

Reply:
I sort of dislike the impact on legibility, wish there was some terse notation to get a symbolic name and check the contents (or the reverse)

Reply:
This is copied from SPDK's code, so I prefer to keep it identical to how SPDK installs liburing: https://github.com/spdk/spdk/blob/master/scripts/pkgdep/common.sh#L6

I'll add a comment.